src/config/config_def.c

```diff
@@ -178,6 +178,11 @@ static const WT_CONFIG_CHECK confchk_WT_CONNECTION_reconfigure[] = {
 	{ NULL, NULL, NULL, NULL, NULL, 0 }
 };
 
+static const WT_CONFIG_CHECK confchk_WT_CONNECTION_rollback_to_stable[] = {
+	{ "timestamp", "string", NULL, NULL, NULL, 0 },
+	{ NULL, NULL, NULL, NULL, NULL, 0 }
+};
```

Comment:
While I'm okay with the current naming, one thought is: can we ever imagine wanting to rollback to any other timestamp?  A more general naming that leaves room for change would be `WT_CONNECTION::rollback_to_timestamp` and `stable_timestamp=` as the API.  That also continues the `stable_timestamp` as a configuration arg more consistent across multiple APIs.